crates/common_utils/src/crypto.rs

```diff
@@ -154,65 +209,103 @@ impl VerifySignature for HmacSha512 {
 /// Represents the GCM-AES-256 algorithm
 #[derive(Debug)]
 pub struct GcmAes256 {
-    nonce: Vec<u8>,
+    // nonce: Vec<u8>,
 }
 
 impl EncodeMessage for GcmAes256 {
     fn encode_message(
         &self,
         secret: &[u8],
         msg: &[u8],
-    ) -> CustomResult<(Vec<u8>, Vec<u8>), errors::CryptoError> {
-        let unbound_key = aead::UnboundKey::new(&aead::AES_256_GCM, secret)
-            .map_err(|_| errors::CryptoError::EncodingFailed)
+    ) -> CustomResult<Vec<u8>, errors::CryptoError> {
+        // let unbound_key = aead::UnboundKey::new(&aead::AES_256_GCM, secret)
+        //     .map_err(|_| errors::CryptoError::EncodingFailed)
+        //     .into_report()
+        //     .attach_printable(RING_ERR_UNSPECIFIED)?;
+
+        // let nonce = aead::Nonce::try_assume_unique_for_key(&self.nonce)
+        //     .map_err(|_| errors::CryptoError::EncodingFailed)
+        //     .into_report()
+        //     .attach_printable(RING_ERR_UNSPECIFIED)?;
+
+        // let sealing_key = aead::LessSafeKey::new(unbound_key);
+        // let mut mutable_msg = msg.to_vec();
+
+        // let tag = sealing_key
+        //     .seal_in_place_separate_tag(nonce, aead::Aad::empty(), &mut mutable_msg)
+        //     .map_err(|_| errors::CryptoError::EncodingFailed)
+        //     .into_report()
+        //     .attach_printable(RING_ERR_UNSPECIFIED)?;
```

Comment:
Will be removing this in the upcoming PRs, added it for reference while modifying the GCM-AES-256 to use randomly generated nonce